js/vendor/sd-streams.d.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @stardazed/streams - implementation of the web streams standard
+ * Part of Stardazed
+ * (c) 2018-Present by Arthur Langereis - @zenmumbler
+ * https://github.com/stardazed/sd-streams
```

Comment:
You can just put at `js/sd_streams.d.ts`

Can you list the license in the header here?